sources/ssh_prompt/local

```diff
@@ -0,0 +1,74 @@
+[ ${SHELL##*/} = "zsh" ] && local GBT__WHICH_OPTS='-p'
+
+function gbt__err() { echo "$@" >&2; }
+
+function gbt_ssh() {
+    [ -z "$GBT__HOME$GBT__PROFILE" ] && gbt__err "'GBT__HOME' not defined" && return 1
+    [ -z "$GBT__HOME$GBT__THEME" ] && gbt__err "'GBT__HOME' not defined" && return 1
+    [ -z "$GBT__PROFILE" ] && GBT__PROFILE="$GBT__HOME/sources/ssh_prompt/remote"
+    [ -z "$GBT__THEME" ] && GBT__THEME="$GBT__HOME/themes/ssh_prompt"
+
+    local SSH_BIN=$(which ${GBT__WHICH_OPTS} ssh 2>/dev/null)
+    [ $? -ne 0 ] && gbt__err "'ssh' not found" && return 1
+
+    if [[ " ${GBT__SSH_IGNORE[*]} " == *" ${@[-1]} "* ]]; then
```

Comment:
Line evaluation fails with following error:
```bash
$ gbt_ssh 192.168.0.1
-bash:  ${@[-1]} : bad substitution
```

I guess `${@[-1]}` is a reference to the last passed argument in `zsh`. Bash equivalent is `${@: -1}`.

As I've mentioned before, to my mind, this check shouldn't be done inside of the ssh function.